app/controllers/photos_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PhotosController < ApplicationController
   def index
     @photos = Photo.all
+    if Cart.last
+    @cart = Cart.last.total_unique_items
```

Comment:
This if statement could also be replaced by something along the lines of:

```ruby
@cart = Cart.last.total_unique_items || 0
```
🤔